### ISSUE TITLE
What's the maximum length for get request ?

### ISSUE THREAD

**Issue by Pierrx56** [dart-lang/http#649]
I want to make some get request and convert it in JSON. It's working pretty well but I don't have the entire response. 
My content length is 1566, is this too large for this library ? 

Thanks

**Comment by brianquinlan** [comment]
No, we have tests for much larger responses. Could you include some example code?
